### PR TITLE
#177 aggregation in output, # 153 BF in output,  #179 BF in exposure 

### PR DESCRIPTION
**Pull request by Santonia27**
Create folder in Output directory for building footprints and aggregation zones/ additional attributes

Create folder in Exposure directory for building footprints

### CODE REVIEW

**Comment by Santonia27** [comment]
Is it possible to have multiple BF?  Currently code set up for a single one. 

If multiple BF can be added I need to add if-statement like in additional attributes.

**Comment by frederique-hub** [comment]
> Is it possible to have multiple BF? Currently code set up for a single one.
> 
> If multiple BF can be added I need to add if-statement like in additional attributes.

Just one is good!!

File: hydromt_fiat/fiat.py [hunk 1]
```diff
@@ -733,6 +736,29 @@ def setup_aggregation_areas(
...
+        # Create additional attributes folder in output
+        additional_att = Path(self.root).joinpath("output" , "additional_attributes")
+        if not os.path.exists(additional_att):
+            os.makedirs(additional_att)
+
+        if isinstance(aggregation_area_fn,list):
+            for file in aggregation_area_fn:
+                shutil.copy2(file, additional_att)
+        else:
+            shutil.copy2(aggregation_area_fn, additional_att)
```
Comment (frederique-hub) [inline hydromt_fiat/fiat.py:749]:
```suggestion
```
Just saving it in the additional_attributes folder in the root folder is sufficient!

File: hydromt_fiat/fiat.py [hunk 2]
```diff
@@ -759,6 +785,25 @@ def setup_building_footprint(
...
+
+        # Create BF folder in Output
+        building_footprints_out = Path(self.root).joinpath("output" , "building_footprints")
+        if not os.path.exists(building_footprints_out):
+            os.makedirs(building_footprints_out)
+        if isinstance(building_footprint_fn,list):
+            for file in building_footprint_fn:
+                shutil.copy2(file, building_footprints_out)
+        else:    
+            shutil.copy2(building_footprint_fn, building_footprints_out)
```
Comment (frederique-hub) [inline hydromt_fiat/fiat.py:806]:
```suggestion
```
It is sufficient to only save the building footprints in the exposure/building_footprints folder

**Review by frederique-hub — CHANGES_REQUESTED** [review]

**Review by frederique-hub — APPROVED** [review]

**Pull request merged**